test/Graphics/Transform.cpp

```diff
@@ -34,7 +34,7 @@ TEST_CASE("sf::Transform class - [graphics]")
             const sf::Transform transform(10.0f, 11.0f, 12.0f,
                                           13.0f, 14.0f, 15.0f,
                                           16.0f, 17.0f, 18.0f);
-            const std::vector<float> matrix(transform.getMatrix(), transform.getMatrix() + 16);
+            const std::vector<float> matrix(transform.getMatrix().begin(), transform.getMatrix().end());
```

Comment:
Can we easily initialize a `const std::array<float, 16>` from this span? I'm not sure if there's a one-line way to do that.

Reply:
As far as I know, there is no trivial way of doing so. That would require a small helper function based on `std::copy` for example.
Alternatively, what we really want here is to compare spans' pointed-to content as if using `std::equal`, but with `operator==` to have a pretty binary check for doctest. I think this is out of scope for this PR but I will be happy to try it later.